spot_rl_experiments/spot_rl/models/owlvit.py

```diff
@@ -214,6 +214,13 @@ def update_label(self, labels):
         labels_with_prefix = [[f"{self.prefix} {label}" for label in labels[0]]]
         self.labels = labels_with_prefix
 
+    def process(self, img, return_image: bool = False):
```

Comment:
Return a dictionary to standardize output too?

Reply:
This would be a bigger change as then we would need to standardize `markrcnn` class too.

maybe this is not needed at this point. Let's deal with this in future when there is a refactor effort to standardize all detectors that we may have in spot-sim2real